exonum/src/node/consensus.rs

```diff
@@ -461,6 +461,13 @@ where
 
         let height = self.state.height();
 
+        let mempool_size = self.state
+            .transactions()
+            .read()
+            .expect("Expected read lock")
+            .len();
+        metric!("node.mempool", "{}", mempool_size);
```

Comment:
Why do we need to format here? Any metric is always just a number. So I prefer to see: 

```rust
metric!("node.mempool", mempool_size);
```